ychat/src/commonMain/kotlin/co/yml/ychat/YChat.kt

```diff
@@ -93,6 +94,21 @@ interface YChat {
      */
     fun imageGenerations(): ImageGenerations
 
+    /**
+     * The edits api is used to edit prompts and re-generate. Given a prompt and an instruction,
+     * the model will return an edited version of the prompt.
+     *
+     * You can configure the parameters of the edits before executing it. Example:
+     * ```
+     * val result = YChat.create(apiKey).edits()
+     *      .setInput("As Descartes said, I think, therefore")
+     *      .setInstruction("Fix spelling mistakes")
+     *      .set...
+     *      .execute()
+     * ```
+     */
+    fun edits(): Edits
```

Comment:
What do you think about adding an integration test for this entrypoint in the YChatTest class?